projects/step-core/src/lib/modules/basics/components/cron-selector/cron-selector.component.html

```diff
@@ -0,0 +1,16 @@
+<step-form-field>
+  <step-label *ngIf="label">{{ label }}</step-label>
+  <input [ngModel]="cronString" (ngModelChange)="valueChange($event)" />
+  <step-suffix class="button-suffix">
+    <button class="btn btn-primary preset-button" mat-button [matMenuTriggerFor]="menu">
```

Comment:
this looks like its using both bootstrap classes and angular material mat-button, can't we get rid of the bootstrap classes?